lib/diego/client.rb

```diff
@@ -168,9 +179,23 @@ def with_request_error_handling(&blk)
       raise RequestError.new(e.message)
     end
 
+    def active_bbs_url
+      uri = URI(base_url)
+      uri.host = "#{active_bbs_id}.#{uri.host}"
+      uri.to_s
+    end
+
+    def active_bbs_id
+      tries ||= 3
+      locket_client.fetch('bbs').resource.owner
+    rescue GRPC::BadStatus => e
```

Comment:
Are these definitely the only errors that could happen? Or should we make this `rescue` more general?

Reply:
Ah that's already the widest error possible - all other the errors in the `grpc` gem inherit from `GRPC::BadStatus`. It inherits from the standard library's `StandardError`.

I guess I could `rescue StandardError` instead, but I'd be concerned we might try to recover from some unrecoverable errors... What do you think?

Reply:
Okay, if that's the "root" GRPC error, that should be fine.